### PR TITLE
Random word gen

### PR DESCRIPTION
**Pull request by ccampbell08**
this code is dope

### CODE REVIEW

File: Dictionary.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+0bigpoops
+1briansucks
```
Comment (blandflakes) [inline Dictionary.txt:2]: This is bullshit

File: HangMan.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 import java.util.ArrayList;
 import java.util.Scanner;
-
+//hi dudes
```
Comment (blandflakes) [inline HangMan.java:16]: Remove useless comment

File: RandomWordGen.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import java.util.stream.Stream;
+import java.lang.Math;
+
+public class RandomWordGen {
+    public static void main(String[] args) throws IOException {
+        int random = (int)(Math.random()*10);
+        String delimiter = Integer.toString(random);
+        Map<String, String> map = new HashMap<>();
+
+        try(Stream<String> lines = Files.lines(Paths.get("/Users/Zeus/Documents/JavaBookClub/Dictionary.txt"))){
```
Comment (blandflakes) [inline RandomWordGen.java:15]: I'd recommend removing the leading numbers and putting each word in an ArrayList<String>. Then just generating a random number between 0 (inclusive) and the length of the list (exclusive) can be used to select and item from the list.

**Review by blandflakes — CHANGES_REQUESTED** [review]

File: RandomWordGen.java (hunk 3 above)
Comment (blandflakes) [reply RandomWordGen.java:15]: To get the lines from a file, you can use Files.readAllLines (from @christopher-brady)

File: Dictionary.txt (hunk 1 above)
Comment (ccampbell08) [reply Dictionary.txt:2]: @blandflakes 
Lol, sorry about this

**Comment by blandflakes** [comment]
All of my pull requests start with "1briansucks"

**Pull request closed without merging**